pkg/server/chaosd/stress.go

```diff
@@ -91,6 +92,10 @@ func (stressAttack) Recover(exp core.Experiment, _ Environment) error {
 	attack := config.(*core.StressCommand)
 	proc, err := process.NewProcess(attack.StressngPid)
 	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
```

Comment:
According to [this comment](https://github.com/golang/go/blob/master/src/os/error.go#L92), it suggests us to use `fs.ErrNotExist` instead of `os.ErrNotExist`.

Reply:
Good! I will use `os.IsNotExist`